### ISSUE TITLE
Android: Reduce Motion/Remove Animations

### ISSUE THREAD

**Issue by amarlette** [facebook/react-native#30871]
## Description
Building any UI that has animations should detect this setting and reduce/remove them accordingly.

## React Native version:
v0.63

**Comment by blavalla** [comment]
This seems to be properly working.

**Comment by waifo** [comment]
Facing same issue as @te-online
AccessibilityInfo.isReduceMotionEnabled is not working properly on android, it's unable to notify if the 'reduce motion' / 'remove animation' is enabled on Android
It works as expected if 'Transition animation scale' value is manually selected as 'Animation off' in Developer options by enabling Developer mode
